css/home.css

```diff
@@ -99,8 +92,65 @@ body {
   font-weight: regular;
 }
 
+
+/* START SERVICES */
+
+.services-container {
+  align-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.service {
+  max-width: 315px;
+  text-align: center;
+  width: 100%;
+}
+
+.service:not(:first-child) {
+  margin-top: 20px;
+}
+
+.service__icon,
+.service__description {
+  display: inline-block;
+}
+
+.service__icon {
+  max-width: 150px;
+}
+
+.service__description {
+  height: 46px;
+  margin: 30px 0 0;
+  max-width: 218px;
+  text-align: center;
+  width: 100%;
+}
+
+/* END SERVICES */
+
+/* START MEDIA QUERIES */
+
 @media only screen and (min-width: 960px) {
```

Comment:
I have some doubts ; in the media query go only the selectors that change for the mobile or you can creates new selector for that. i think that you only create all in the mobile area and later you changes the specs for de desktop area but not create new ones. You can explain that, please?

Reply:
it is not the usual thing, but you can have a selector that does absolutely nothing in a viewport and does have some rules in other.

Reply:
ok.